doc/shelley-genesis.md

```diff
@@ -506,13 +506,133 @@ if you want.
 
 ## Creating node operational keys
 
-TODO
+In order to run a shelley node we need to generate:
+
+- KES Keys
+- VRF Keys
+- An operational certificate
+
+#### KES Keys
+
+```
+cabal exec cardano-cli -- shelley node key-gen-KES
+                        --verification-key-file example/KES/verkey
+                        --signing-key-file example/KES/signkey
+```
+
+#### VRF Keys
+
+```
+cabal exec cardano-cli -- shelley node key-gen-VRF
+                        --verification-key-file example/VRF/verkey
+                        --signing-key-file example/VRF/signkey
+```
+
+#### Operational Certificate
+
+```
+cabal exec cardano-cli -- shelley node issue-op-cert
+                        --hot-kes-verification-key-file example/KES/verkey
+                        --cold-signing-key-file example/delegate-keys/delegate1.skey
+                        --operational-certificate-issue-counter example/delegate-keys/delegate-opcert1.counter
+                        --kes-period 0
+                        --out-file example/opcert
+```
+
+
 
 ## Issuing node operational certificates
 
 TODO
 
 ## Starting a node
 
-TODO
+Now that we have generated our genesis.json, operational certificate, VRF signing key and KES signing key, we can run a node via:
+--TODO: What about creation of topology?
+--TODO: Probably need to add a default config.yaml for a shelley node
+```
+cabal exec cardano-node -- run
```

Comment:
```suggestion
$ cardano-node -- run
```
We said at the start of the doc:
> This assumes that you already have built and installed the executables from
> this repository. So `cardano-cli` and `cardano-node` should be on your `$PATH`.

So we should follow that pattern, and not mention `cabal`.